.github/workflows/fuzz.yml

```diff
@@ -125,6 +125,14 @@ jobs:
       claude_code_oauth_token: ${{ secrets.CLAUDE_CODE_OAUTH_TOKEN }}
       gh_token: ${{ secrets.GITHUB_TOKEN }}
 
+  attempt-fix-io:
+    name: "Attempt Fix for IO Fuzz Crash"
+    needs: report-io-fuzz-failures
+    if: always() && needs.report-io-fuzz-failures.outputs.issue_number != ''
```

Comment:
Hah, what is even going on here: ` if: always() && needs.report-io-fuzz-failures.outputs.issue_number != ''`